### DISCUSSION TITLE
What's with the comma on multi-day events?

### DISCUSSION THREAD

**Discussion by variada** [totaldebug/atomic-calendar-revive#1192] [Support]
Hi,

just a quick question. It's probably not a bug as such, it just irks me slightly when using the card.

In event view, if there's a multi-days event, the information line always seems to start with a comma on our card.

like this:
![Screenshot 2023-10-02 114738](https://github.com/totaldebug/atomic-calendar-revive/assets/83453466/d2fc2a9c-ee24-49c5-a882-126f790428d5)

Background information: I usually hide location and description, but tried setting and showing them both. They don't fill whatever is missing there. it's not breaking the use of the card, but it looks strange as it is. 

so question is: what bit of information am I missing there? or: is there any way to just get: until xy. without that comma?

**Comment by variada** [comment]
Found out by accident.

![Screenshot 2023-10-04 122618](https://github.com/totaldebug/atomic-calendar-revive/assets/83453466/96962129-4688-48ed-bccd-4c5edf222baa)

It's the "All Day" text that's missing. as I usually don't show that (every event with time is timed, everything else is all day).

so, is it possible to get rid of that unwanted comma if the "All Day" text is disabled?

**Reply by marksie1988** [reply]
@variada would you be able to provide a copy of your configuration to assist with looking at resolving this.
